src/components/MoneyReportHeader.tsx

```diff
@@ -182,6 +189,105 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
         [moneyRequestReport, chatReport, policy, transaction],
     );
 
+    const [isDownloadErrorModalVisible, setIsDownloadErrorModalVisible] = useState(false);
+
+    const {selectedTransactionsID, setSelectedTransactionsID} = useMoneyRequestReportContext(moneyRequestReport?.reportID);
+
+    const selectedTransactionsOptions = useMemo(() => {
+        if (!selectedTransactionsID.length) {
+            return [];
+        }
+        const options = [];
+        const selectedTransactions = selectedTransactionsID.map((transactionID) => getTransaction(transactionID)).filter((t) => !!t);
+
+        const anyTransactionOnHold = selectedTransactions.some(isOnHoldTransactionUtils);
+        const allTransactionOnHold = selectedTransactions.every(isOnHoldTransactionUtils);
+
+        if (!anyTransactionOnHold && selectedTransactions.length === 1) {
```

Comment:
Why was not the hold option shown when bulk selecting? Adding this line caused https://github.com/Expensify/App/issues/60111

Reply:
I can't find conversation on slack because it was discussed in long thread but it was agreed that we don't want to iterate over selected issues and send many requests. The go to solution was to temporarily disable that option if many transactions are selected and add it when there's api to hold many transaction in one request.

Reply:
We have a `BulkHoldRequest` API in place, so we can remove this limitation and start working on a fix for that.

Reply:
@luacmartins great! Is there issue for it? I can replace it

Reply:
I don't think we have an issue yet, but I think @Tony-MK might be working on that